exams/api.py

```diff
@@ -75,6 +75,14 @@ def authorize_for_exam_run(mmtrack, course_run, exam_run):
         course_run (courses.models.CourseRun): A CourseRun object.
         exam_run (exams.models.ExamRun): the ExamRun we're authorizing for
     """
+    if not mmtrack.user.is_active:
```

Comment:
In each other case we raise `ExamAuthorizationException` in this function. Can we do the same here?